### PR TITLE
IRI Terminology

### PR DESCRIPTION
**Pull request by gkellogg**
For #15.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/rdf-concepts/pull/17.html" title="Last updated on Feb 20, 2023, 6:59 PM UTC (8912f1d)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/rdf-concepts/17/e93ecf0...8912f1d.html" title="Last updated on Feb 20, 2023, 6:59 PM UTC (8912f1d)">Diff</a>

### CODE REVIEW

**Comment by afs** [comment]
Do not suggest in any way that the WG may change core terminology without the WG having agreed to that being said.

**Comment by afs** [comment]
There is no mention of "adopt the pertinent core of RFC3987".

File: spec/index.html [hunk 1]
```diff
@@ -492,9 +492,13 @@ <h3>IRIs</h3>
...
-        [[RFC3987]] never made it past the Proposed Standard phase
-        and should not be normatively cited.
+      <p class="issue" data-number="15">
+        A future version of this specification may use [[?URL]],
+        as [[?RFC3987]] is an abandoned specification.
+        This would include retaining most of the `IRI`-based nomenclature, but
+        defining in terms of
+        <a data-cite="URL#absolute-url-with-fragment-string">absolute-URL with fragment string</a>
+        and <a data-cite="?URL#relative-url-string">relative-URL string</a>.
       </p>
```
Comment (TallTed) [inline spec/index.html:502]: It seems this is all obviated by [this comment on #15](https://github.com/w3c/rdf-concepts/issues/15#issuecomment-1436613178)

**Comment by gkellogg** [comment]
> Do not suggest in any way that the WG may change core terminology to URL without the WG having agreed to that being said.

In re-creating the PR, this seems to have lost the more neutral text change you previously requested, I'll update.

**Comment by gkellogg** [comment]
Closing in light of Martin's comment https://github.com/w3c/rdf-concepts/issues/15#issuecomment-1436613178.

File: spec/index.html [hunk 2]
```diff
@@ -493,12 +493,8 @@ <h3>IRIs</h3>
...
 
       <p class="issue" data-number="15">
-        A future version of this specification may use [[?URL]],
-        as [[?RFC3987]] is an abandoned specification.
-        This would include retaining most of the `IRI`-based nomenclature, but
-        defining in terms of
-        <a data-cite="URL#absolute-url-with-fragment-string">absolute-URL with fragment string</a>
-        and <a data-cite="?URL#relative-url-string">relative-URL string</a>.
+        The group is aware that work has not progressed on [[RFC3987]],
+        and is considering alternatives.
```
Comment (afs) [inline spec/index.html:497]: Given Martin's comments `<p>` can be removed.

#15 is quite limied -- "Revise terminology to align with current RFCs" from [PA's observation of incorrect use of "relative IRI"](https://lists.w3.org/Archives/Public/public-rdf-comments/2021Mar/0002.html). 

If you want something :

```suggestion
        The group is updating IRI terminology to more closer follow the current RFCs.
```

**Review by afs — APPROVED** [review]

**Comment by gkellogg** [comment]
@rdfguy, @ktk: in spite of the procedural issues about appropriate work items, I think this PR should be merged, as the existing text is now inappropriate.

Note that this PR simply updates the issue marker to clarify that the open work item is on [erratum 29 Terminology about IRIs](https://www.w3.org/2001/sw/wiki/RDF1.1_Errata#erratum_29) not on the suitability of using RFC3987.

**Comment by afs** [comment]
[Relates to #21.](https://github.com/w3c/rdf-concepts/pull/21#pullrequestreview-1338634092)

**Comment by gkellogg** [comment]
It's worse to leave this PR unmerged than to keep the status quo, so merging.

**Pull request merged**